src/core/jupyter/jupyter-fixups.ts

```diff
@@ -165,9 +165,15 @@ export function fixupFrontMatter(nb: JupyterNotebook): JupyterNotebook {
     }
   });
 
-  // if we have front matter and a title then we are done
   const yaml = partitioned ? readYamlFromMarkdown(partitioned.yaml) : undefined;
-  if (yaml?.title) {
+  if (
+    // if we have front matter and a title then we are done
+    yaml?.title ||
+    // if we have front matter and it has revealjs as a format then we are done too
+    (yaml?.format !== null &&
```

Comment:
I think the idea of this is that a python user with existing notebooks may want to render them using `quarto render foo.ipynb --to html` or similar commands and get a reasonable experience. Since it is common to place a heading as the 'title' of the notebook in a markdown cell at the top of the notebook, we're trying to find the best way to promote that to the document title in Quarto.

It's pretty tempting to just that if the user creates front matter at all, they should place a title there (not just for revealJS - just that once they're making front matter we expect them to handle the title)...

We could start with this if you're feeling conservative.

Reply:
The revealjs specific case is that 

- Using `title` will trigger the use of a specific `title-slide` sectionf from the revealjs template
- To opt out this title slide from the template, no providing `title` in YAML is for now the way to go. 

Currently it seems you can't do that with `engine: jupyter`, or at least that we promote headers like `##`  without considering the `slide-level`, meaning that a non section slide could be promoted to section slide. 

This is something quite specific to revealjs here related to this heading promoting feature. 🤔